src/sidebar/store.js

```diff
@@ -143,6 +143,7 @@ function store($http, $q, auth, settings) {
       get: apiCall('annotation.read'),
       update: apiCall('annotation.update'),
     },
+    flag: apiCall('flag.create'),
```

Comment:
For consistency this should probably be `flag: { create: apiCall('flag.create') }`.